src/diffusers/pipelines/__init__.py

```diff
@@ -1,460 +1,473 @@
-from typing import TYPE_CHECKING

```

Comment:
git diff does a horrible job here as the __init__ is not changed that much. I checked it manually and it should be good